mmdet/datasets/pipelines/loading.py

```diff
@@ -213,6 +213,9 @@ class LoadAnnotations:
             annotation. Default: False.
         poly2mask (bool): Whether to convert the instance masks from polygons
             to bitmaps. Default: True.
+        normed_bbox (bool): Whether to convert bbox from relative value to
```

Comment:
Can this part be done in `dataset.get_ann_info`? Just like CocoDataset converts xywh to xyxy in `_parse_ann_info`

Reply:
seems cannot, because we cannot get `h` and `w` form annotation file.